code/src/java/pcgen/gui2/facade/Gui2InfoFactory.java

```diff
@@ -1943,6 +1930,251 @@ public String getDescription(AbilityFacade ability)
 	}
 
 	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public String getDescription(RaceFacade raceFacade)
+	{
+		if (raceFacade == null || !(raceFacade instanceof Race))
+		{
+			return EMPTY_STRING;
+		}
+		try
+		{
+			Race race = (Race) raceFacade;
+			return pc.getDescription(race);
```

Comment:
These should all use the DescriptionFormatting.piWrapDesc method to ensure that product identity names are correctly formatted